content/en/account_management/billing/custom_metrics.md

```diff
@@ -7,64 +7,224 @@ aliases:
+
+The billable number of custom metrics is based on the average number of custom metrics (from all paid hosts) per hour over a given month.
+These allocations are counted across your entire infrastructure. For example, if you are on the Pro plan and licensed for three hosts, 300 custom metrics are allocated. The 300 custom metrics can be divided equally across each host, or all 300 metrics could be used by a single host.
+
+Using this example, the graphic below shows scenarios that do not exceed the allocated custom metric count:
+
+{{< img src="account_management/billing/custom_metrics/host_custom_metrics.png" alt="host_custom_metrics" responsive="true" >}}
+
+Contact [Sales][3] or your [Customer Success][4] Manager to discuss custom metrics for your account or purchase an additional custom metrics packages.
+
+## Tracking Custom metrics
+
+Administrative users (those with [Datadog Admin roles][5]) can see the total custom metrics per hour and the top 500 custom metrics for their account on the [usage details page][6]. See the [Usage Details][7] documentation for more information.
+
+For more real-time tracking of the count of custom metrics for a particular metric name, click into the metric name on the [Metrics Summary page][8]; it’s listed as “Currently reporting # distinct metrics…” as shown below:
+
+{{< img src="account_management/billing/custom_metrics/tracking_metric.mp4" alt="Tracking metric" video="true" responsive="true">}}
+
+## Counting Custom Metrics
+
+**A custom metric is uniquely identified by a combination of a metric name and tag values (including the host tag)**:
+
+* Reporting the same metric name on multiple hosts results in multiple custom metrics.
+* Adding tags on a metric can change the number of custom metrics (number of unique tag value combinations) associated with that particular metric. See the [Effect of Adding Tags](#effect-of-adding-tags) section.
+* Reordering of tag values doesn’t add uniqueness, the following combinations are the same custom metric:
+
+  *  `metric_name{env:dev, service:foo}`
+  *  `metric_name{service:foo, env:dev}`
+
+Find below some example of how to count your custom metrics. The number of custom metrics associated with a particular metric name depends on its metric [submision type][9]:
+
+{{< tabs >}}
+{{% tab "Count, Rate, Gauge" %}}
+
+The number of custom metrics from [COUNT][1], [RATE][2], and [GAUGE][3] metric types is calculated the same way as shown below:
+
+Suppose you’re submitting a metric, `request.count`, from two hosts (`host:A`,`host:B`), which counts the number of endpoint requests. You’re submitting this metric with two tag keys:
+
+* `endpoint`  that can take the value `endpoint:X` or `endpoint:Y`
+* `status` that can take the value `status:200` or `status:400`
+
+Let’s assume that in your data, endpoint `endpoint:X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:
```

Comment:
```suggestion
Let’s assume that your data endpoint `endpoint:X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:
```

Reply:
this change obfuscates the meaning of the example, 'data endpoint' and 'in your data, `"endpoint:X"` are different. 

Let's proceed with the following edit: 
"Let’s assume that in your data, `endpoint:X` is supported by both hosts, but fails only on `host:B`, and requests to `endpoint:Y` are always successful and only appears on `host:B` as shown below:"

Reply:
👍